src/Altinn.App.Core/Features/Options/AppOptionsService.cs

```diff
@@ -9,14 +9,16 @@ public class AppOptionsService : IAppOptionsService
     {
         private readonly AppOptionsFactory _appOpptionsFactory;
         private readonly InstanceAppOptionsFactory _instanceAppOptionsFactory;
+        private readonly TableAppOptionsFactory _tableAppOpptionsFactory;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="AppOptionsService"/> class.
         /// </summary>
-        public AppOptionsService(AppOptionsFactory appOptionsFactory, InstanceAppOptionsFactory instanceAppOptionsFactory)
+        public AppOptionsService(AppOptionsFactory appOptionsFactory, InstanceAppOptionsFactory instanceAppOptionsFactory, TableAppOptionsFactory tableAppOptionsFactory)
```

Comment:
Since it's not related to options we should create a new service and not extend the AppOptionsService/IAppOptionsService.

Reply:
Or perhaps, IAppListsService just not to confuse it with IList/List in .Net